send-emails-from-a-service/README.md

```diff
@@ -0,0 +1,681 @@
+        "cardNo": "7844481124110331"
+    },
+    "doctor": "thomas collins",
+    "hospital_id": "grandoaks",
+    "hospital": "grand oak community hospital",
+    "appointment_date": "2023-10-02"
+}
+```
+
+2. Extract necessary details from the request (e.g., hospital, patient, doctor, etc.) and make a call to the hospital backend service to request an appointment. A response similar to the following will be returned from the hospital backend service on success.
+
+```json
+{
+    "appointmentNumber": 1,
+    "doctor": {
+        "name": "thomas collins",
+        "hospital": "grand oak community hospital",
+        "category": "surgery",
+        "availability": "9.00 a.m - 11.00 a.m",
+        "fee": 7000.0
+    },
+    "patient": {
+        "name": "John Doe",
+        "dob": "1940-03-19",
+        "ssn": "234-23-525",
+        "address": "California",
+        "phone": "8770586755",
+        "email": "johndoe@gmail.com"
+    },
+    "hospital": "grand oak community hospital",
+    "confirmed": false,
+    "appointmentDate": "2023-10-02"
+}
+```
+
+3. Finally, call the payment backend service to make the payment and retrieve the reservation response. Send an email to the user with the appointment details.
+
+```json
+{
+    "appointmentNo": 1,
+    "doctorName": "thomas collins",
+    "patient": "John Doe",
+    "actualFee": 7000,
+    "discount": 20,
+    "discounted": 5600.0,
+    "paymentID": "8458c75a-c8e0-4d49-8da4-5e56043b1a20",
+    "status": "settled"
+}
+```
+
+### Concepts covered
+
+- REST API
+- HTTP Client
+- Email Client
+
+## Develop the application
+
+### Step 1: Set up the workspace
+
+Install [Ballerina Swan Lake](https://ballerina.io/downloads/) and the [Ballerina Swan Lake VS Code extension](https://marketplace.visualstudio.com/items?itemName=wso2.ballerina) on VS Code.
+
+### Step 2: Develop the service
+
+Follow the instructions given in this section to develop the service.
+
+1. Create a new Ballerina project using the `bal` command and open it in VS Code.
+
+```bash
+$ bal new send-emails-from-a-service
+```
+
+2. Introduce the source code in files with the `.bal` extension (e.g., the `main.bal` file). 
+
+Import the 
+- `ballerina/email` module to send emails using the SMTP protocol
+- `ballerina/http` module to develop the REST API and define the clients that can be used to send requests to the backend services
+- `ballerina/log` module to log debug/error information for each client request
+
+```ballerina
+import ballerina/email;
+import ballerina/http;
+import ballerina/log;
+```
+
+3. Define [configurable variables](https://ballerina.io/learn/by-example/#configurability) for the port on which the listener should listen and the URLs of the backend services. Also define configurable variables for the host, username, and password of the SMTP client. Enable two factor authentication on your google account, create an app password and use that in place of your true email password.
```

Comment:
Why can't we combine  ` Define [configurable variables](https://ballerina.io/learn/by-example/#configurability) for the port on which the listener should listen and the URLs of the backend services` and `Also define configurable...`? I feel we try to say the same thing.